src/blocks/select/Select.tsx

```diff
@@ -219,19 +219,10 @@ const Select: React.FC<SelectProps> = ({
   const parentRef = useRef<HTMLDivElement>(null);
   const childRef = useRef<HTMLInputElement>(null);
 
-  // useEffect(() => {
-  //   const handleResize = () => {
-  //     if (comboboxRef.current) {
-  //       setPopoverWidth(comboboxRef.current.offsetWidth - 18);
-  //     }
-  //   };
+  useEffect(() => {
+    window.addEventListener('scroll', () => setViewPopover(false));
```

Comment:
@mishramonalisha76 we also need to remove the listener on component unmount.

Reply:
done